src/raftstore/store/region_snapshot.rs

```diff
@@ -785,4 +902,95 @@ mod tests {
         res.sort();
         assert_eq!(res, test_data[1..3].to_vec());
     }
+
+    #[test]
+    fn test_delete_files_in_range() {
```

Comment:
Add some inline comment the explain the test? Like at which point which blob file will get GC-ed, which key will be left in the DB if we hit the previous Titan bug, etc.